catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -71,6 +79,54 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
   const { isFullSupportBrowser } = useContext(AppContext);
   const [expandedImage, setExpandedImage] = useState<Image | undefined>();
   const [isActive, setIsActive] = useState(false);
+  const router = useRouter();
+
+  const getImage = async routerImageId => {
+    const apiUrl = `https://api.wellcomecollection.org/catalogue/v2/images/${routerImageId}`;
+    const image: Image = await fetch(apiUrl).then(res => res.json());
+
+    if (image) {
+      setExpandedImage(image);
+      setIsActive(true);
+    }
+  };
+
+  const encodeDecodeImageIdInUrl = (mode: 'encode' | 'decode') => {
+    const searchTest = window.location.search;
```

Comment:
I'm not sure about using a query parameter to hold the ID - it feels like a fragment (#) could also work and be more appropriate? That is [what they are intended to be used for](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Identifying_resources_on_the_Web#fragment)